README.rst

```diff
@@ -130,16 +130,35 @@ the testsuite with the following command:
 
 .. code-block:: bash
 
-        python setup.py test
+    python setup.py test
 
 Otherwise using a virtualenv is recommended. Setuptools will automatically fetch
 missing test dependencies:
 
 .. code-block:: bash
 
-        virtualenv
-        source .venv/bin/active
-        (venv) python setup.py test
+    virtualenv
+    source .venv/bin/active
+    (venv) python setup.py test
+
+Development
+-----------
+
+* Make sure that you have Java_ installed.
```

Comment:
For running `jenkinsapi_tests/systest` you should launch java-file https://github.com/pycontribs/jenkinsapi/blob/master/jenkinsapi_utils/jenkins_launcher.py#L158